### ISSUE TITLE
Custom Directives: some attributes are broken (as well as attrs with dashes) [Accessibility]

### ISSUE THREAD

**Issue by mpicard** [yyx990803/vue#958]
v0.12.1:

As you can see in the fiddle provided, some attributes such as class, role, etc do not work properly. As well as attributes with dashes like aria-valuemin, aria-valuemax, since it is an invalid left assignment, using a string (eg. this.el.'aria-valuemin') also does not work. This obviously is a problem for accessibility.

http://jsfiddle.net/4c77uxyb/

**Comment by mpicard** [comment]
oops this.el.setAttributes(...) is the proper method

https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute
